### PR TITLE
Add homepage

### PR DESCRIPTION
**Pull request by chidiebereojingwa**
## Portfolio website
my portfolio website.
built with 
- html
- css
- linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+HTML
+CSS
+JavaScript
+### Setup
+clone to you local machine
+create a directory on a your local machin.
+open your terminal and type: git clone https://github.com/chidiebereojingwa/Hello_Microverse.git
+### Usage
+Clone project to your local machine 
+run the index.html
```
Comment (george-shammar) [inline README.md:20]: - [ ] Please, update your README with the missing sections e.g `LICENCE` section. I encourage you to use the README template provided by Microverse and customize the content.
- [ ] Please improve the Pull Request message. Typically, it should be the summary of the features you implemented.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        <div class="social">
+            <a href="#" class="fa fa-github"></a>
+            <a href="#" class="fa fa-linkedin"></a>
+            <a href="#" class="fa fa-angellist"></a>
+            <a href="#" class="fa fa-twitter"></a>
+            <a href="#" class="fa fa-medium"></a>
+        </div>
+    </div>
+
```
Comment (george-shammar) [inline index.html:40]: - [ ] The list of icons uses a `ul` tag to group them, and` li` per icon; each icon has a link (`a tag`). Please, let's update that. :pray:

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @chidiebereojingwa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello, @chidiebereojingwa great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**